src/app/views/query-runner/request/permissions/Permissions.QueryV9.tsx

```diff
@@ -0,0 +1,170 @@
+import {
+  Table, TableHeader, TableRow, TableCell, TableBody,
+  Label, makeStyles
+} from '@fluentui/react-components';
+import { useContext, useEffect, useState } from 'react';
+import { useAppDispatch, useAppSelector } from '../../../../../store';
+import { IPermission } from '../../../../../types/permissions';
+import { ValidationContext } from '../../../../services/context/validation-context/ValidationContext';
+import { usePopups } from '../../../../services/hooks';
+import { fetchAllPrincipalGrants } from '../../../../services/slices/permission-grants.slice';
+import { fetchScopes } from '../../../../services/slices/scopes.slice';
+import { ScopesError } from '../../../../utils/error-utils/ScopesError';
+import { translateMessage } from '../../../../utils/translate-messages';
+import { convertVhToPx } from '../../../common/dimensions/dimensions-adjustment';
+import { getColumns } from './columnsV9';
+import { setConsentedStatus, sortPermissionsWithPrivilege } from './util';
+import { FontSizes, Link } from '@fluentui/react';
+
+const useStyles = makeStyles({
+  root: {
+    padding: '17px'
+  },
+  label: {
+    marginLeft: '12px'
+  },
+  errorLabel: {
+    marginTop: '10px',
+    paddingLeft: '10px',
+    paddingRight: '20px',
+    minHeight: '200px'
+  },
+  permissionText: {
+    fontSize: FontSizes.small,
```

Comment:
hmm, v8 token. Use `<Text>some text</Text> in v9 and set the size you want https://react.fluentui.dev/?path=/docs/components-text--docs#size